datasets/README.md

```diff
@@ -0,0 +1,23 @@
+# Datasets
+
+This directory contains the conversational semantic parsing datasets we used for the experiments of the following paper:
+
+```bibtex
+@inproceedings{SMValueAgnosticParsing2020,
+  author = {Platanios, Emmanouil Antonios and Pauls, Adam and Roy, Subhro and Zhang, Yuchen and Kyte, Alex and Guo, Alan and Thomson, Sam and Krishnamurthy, Jayant and Wolfe, Jason and Andreas, Jacob and Klein, Dan},
+  title = {Value-Agnostic Conversational Semantic Parsing},
+  booktitle = {Proceedings of the 59th Annual Meeting of the Association for Computational Linguistics},
+  month = aug,
+  year = {2021},
+  address = {Online},
+  publisher = {Association for Computational Linguistics},
+}
+```
+
+There are two datasets:
+
+- **SMCalFlow 2.0:** This is an updated version of the dataset released with the [**Task-Oriented Dialogue as Dataflow Synthesis** (TACL 2020)](https://www.mitpressjournals.org/doi/full/10.1162/tacl_a_00333) paper, which removed a very small number of incorrectly annotated examples, dropped argument names for positional arguments (so that the programs are shorter), and added inferred type arguments for type-parameterized functions that were missing in the original SMCalFlow data.
+- **TreeDST:** This is a modified version of the [TreeDST dataset]([apple/ml-tree-dst (github.com)](https://github.com/apple/ml-tree-dst)) which has been converted to the Lispress representation used for SMCalFlow 2.0, and transformed to make use of the `refer` and `revise` meta-computation operators. The transformation is described in the appendix of the paper referenced above.
```

Comment:
> [TreeDST dataset]([apple/ml-tree-dst (github.com)](https://github.com/apple/ml-tree-dst))

Is this format intended? I thought you may want to use `[TreeDST dataset](https://github.com/apple/ml-tree-dst)` but I'm not 100% sure.

Reply:
No it's accidental because Edge copies urls like that now by default. Good catch.